[rospy.client][INFO] 2021-10-22 12:19:19,519: init_node, name[/ss42/decoder_node], pid[14043]
[xmlrpc][INFO] 2021-10-22 12:19:19,520: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2021-10-22 12:19:19,541: Started XML-RPC server [http://ss42.local:44068/]
[rospy.init][INFO] 2021-10-22 12:19:19,542: ROS Slave URI: [http://ss42.local:44068/]
[rospy.impl.masterslave][INFO] 2021-10-22 12:19:19,543: _ready: http://ss42.local:44068/
[rospy.registration][INFO] 2021-10-22 12:19:19,547: Registering with master node http://ss42.local:11311/
[xmlrpc][INFO] 2021-10-22 12:19:19,548: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-10-22 12:19:19,644: registered with master
[rospy.rosout][INFO] 2021-10-22 12:19:19,645: initializing /rosout core topic
[rospy.rosout][INFO] 2021-10-22 12:19:19,657: connected to core topic /rosout
[rospy.simtime][INFO] 2021-10-22 12:19:19,663: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-10-22 12:19:19,744: [/ss42/decoder_node] ~publish_freq = 2.0 
[rospy.internal][INFO] 2021-10-22 12:19:19,822: topic[/ss42/camera_node/image/compressed] adding connection to [http://ss42.local:44402/], count 0
[rospy.internal][INFO] 2021-10-22 12:19:19,910: topic[/ss42/camera_node/image/raw] adding connection to [/ss42/raw_cam_info_reader_node], count 0
[rospy.internal][INFO] 2021-10-22 12:19:19,929: topic[/rosout] adding connection to [/rosout], count 0
[rosout][WARNING] 2021-10-22 12:19:20,617: Inbound TCP/IP connection failed: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.
[rospy.internal][ERROR] 2021-10-22 12:19:20,628: Inbound TCP/IP connection failed:
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 321, in _tcp_server_callback
    header = read_ros_handshake_header(sock, StringIO(), buff_size)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosgraph/network.py", line 364, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.internal][ERROR] 2021-10-22 12:19:20,673: Unable to initiate TCP/IP socket to ss42.local:42011 (http://ss42.local:44402/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2021-10-22 12:19:20,837: topic[/ss42/camera_node/image/raw] removing connection to /ss42/raw_cam_info_reader_node
[rospy.internal][WARNING] 2021-10-22 12:19:21,681: Unknown error initiating TCP/IP socket to ss42.local:42011 (http://ss42.local:44402/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2021-10-22 12:19:21,682: topic[/ss42/camera_node/image/compressed] removing connection to http://ss42.local:44402/
[rospy.internal][INFO] 2021-10-22 12:19:21,708: topic[/ss42/camera_node/image/raw] adding connection to [/ss42/raw_cam_info_reader_node], count 0
[rospy.internal][INFO] 2021-10-22 12:19:21,839: topic[/rosout] removing connection to /rosout
[rospy.core][INFO] 2021-10-22 12:19:30,429: signal_shutdown [signal-2]
[rospy.registration][WARNING] 2021-10-22 12:19:30,436: cannot unregister with master due to network issues
[rospy.internal][INFO] 2021-10-22 12:19:30,438: topic[/ss42/camera_node/image/raw] removing connection to /ss42/raw_cam_info_reader_node
[rospy.impl.masterslave][INFO] 2021-10-22 12:19:30,440: signal-2
[rospy.core][INFO] 2021-10-22 12:19:30,574: signal_shutdown [atexit]
